plugin/package.json

```diff
@@ -11,8 +11,7 @@
   "license": "MIT",
   "readmeFilename": "README.md",
   "scripts": {
-    "prepare": "yarn generate",
-    "generate": "yarn tsc && yarn bundle && yarn format && yarn clean",
+    "build": "yarn tsc && yarn bundle && yarn format && yarn clean",
```

Comment:
Please make sure that we don't have `yarn generate` anywhere left prior to merging.